src/components/share-structure/ShareStructure.vue

```diff
@@ -38,7 +38,7 @@
       <div class="btn-container py-6 px-4">
         <v-btn
           id="btn-add-person"
-          outlined
+          variant="outlined"
```

Comment:
since `flat / outlined / text / plain props have been combined into a single variant prop.`
https://vuetifyjs.com/en/getting-started/upgrade-guide/#v-btn-v-btn-toggle